### ISSUE TITLE
Tests 000_dihedralPC_abf2d and 018_pathCV are failing

### ISSUE THREAD

**Issue by giacomofiorin** [Colvars/colvars#275]
The differences are likely due to recent changes in the extended Lagrangian:
```
cat 000_dihedralPC_abf2d/test.colvars.state.stripped.diff
9c9
<   extended_x -1.13949025362870e+00
---
>   extended_x -1.14502810666269e+00
10c10
<   extended_v  5.53785303399100e-03
---
>   extended_v  4.95564329068078e-03
16c16
<   extended_x -2.11503223146226e+00
---
>   extended_x -2.11383784748319e+00
17c17
<   extended_v -1.19438397907173e-03
---
>   extended_v -1.36281501667054e-03
```

**Comment by jhenin** [comment]
![1o0gxyrerkd31](https://user-images.githubusercontent.com/4518166/62230365-f27f4d80-b3c1-11e9-845b-38e5901f8732.jpg)

**Comment by giacomofiorin** [comment]
Closed in #282
